thingsvision/core/cka/cka_numpy.py

```diff
@@ -59,13 +59,15 @@ def linear_kernel(self, X: Array) -> Array:
         """Use a linear kernel for computing the gram matrix."""
         return X @ X.T
 
-    def rbf_kernel(self, X: Array, sigma: float = 1.0) -> Array:
+    def rbf_kernel(self, X: Array) -> Array:
         """Use an rbf kernel for computing the gram matrix. Sigma defines the width."""
         GX = X @ X.T
         KX = np.diag(GX) - GX + (np.diag(GX) - GX).T
-        if sigma is None:
+        if self.sigma is None:
             mdist = np.median(KX[KX != 0])
             sigma = math.sqrt(mdist)
```

Comment:
Question, if self.sigma=None, you compute it's value, but shall it be stored? Or shall we recompute it every time?

Reply:
Ok, agree